components/git/land.js

```diff
@@ -59,7 +61,7 @@ const CONTINUE = 'continue';
 const ABORT = 'abort';
 
 function handler(argv) {
-  if (argv.prid && Number.isInteger(argv.prid)) {
+  if (argv.prid && (Number.isInteger(argv.prid) || isUrl(argv.prid))) {
```

Comment:
We don't really need this module, do we? We can simply use a regexp to validate & capture the id: `argv.prid.match(/github.com/\w+/\w+/pull/(\d+)/)`

Reply:
@joyeecheung makes sense, also helps having less direct dependencies as they are 👍. making changes in few